.github/workflows/ci.yml

```diff
@@ -0,0 +1,55 @@
+name: CI
+on:
+  push:
+  pull_request:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 1 * *'  # Monthly
+
+env:
+  CHILDPROCESS_UNSET: should-be-unset
+
+jobs:
+
+  test:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        ruby-version: ['2.4', '2.5', '2.6', '2.7', '3.0', jruby-9.2]
+        posix_spawn: [true, false]
+        exclude:
+          # The action used seems to not fully support JRuby on Windows
```

Comment:
What's the issue? The seems to imply it's an issue of ruby/setup-ruby, but I don't of any.

Reply:
I think that that build combination just hung forever trying to even start up. No getting to the tests and failing just a lot of "why is this job still spinning and having no output after over an hour?". Although, now that I go back and look at the cancelled build in order to link it, it now has output. Still took me killing the process for that to happen, though.

https://github.com/enkessler/childprocess/runs/2755087269?check_suite_focus=true

Reply:
That seems a bug of GitHub Actions then, nothing to with `ruby/setup-ruby` (which I guess is meant by `The action`).

Reply:
A bit odd that a GitHub Action bug would only appear for JRuby but not for the other flavors, given that they all use the same underlying action. That could be the case, though.

Regardless, I just chucked it into the "stuff that I'm not going to worry about right now" bucket because there was no obvious reason for it to not work, that action not necessarily working for JRuby+Windows is a [known issue](https://github.com/ruby/setup-ruby/issues/18), and not having JRuby + Windows build combination wasn't any worse than our current CI situation. If you want to try throwing JRuby back into the mix and seeing what happens, feel free to do so.

Reply:
Yeah, it's probably fine to ignore for now.
What I meant as a ruby/setup-ruby maintainer is I believe this issue has nothing to do with ruby/setup-ruby.
I guess it's either a bug of JRuby on Windows (e.g., the issue you linked), or of GitHub Actions.

Reply:
Oh. In that case, yeah, you'd have a better idea than most what the problem might be. ;)

Hello famous person. *waves*